app/models/user.rb

```diff
@@ -4,4 +4,5 @@ class User < ApplicationRecord
   devise :database_authenticatable, :registerable,
          :recoverable, :rememberable, :validatable
   has_many :comments
+  has_many :carts
```

Comment:
Hm, can a user really have many carts concurrently?  That seems confusing to me.  Most applications only allow one cart at a time.

Reply:
I didn't think of it in the sense of one cart at a time, I was just thinking of it in the sense that a user can have many carts over time. If it makes more sense, I can change this to a one to one relationship instead.